### PR TITLE
fix(editor-packages): Use one redux store per instance of editor

### PR DESCRIPTION
**Pull request by CodingDive**

### CODE REVIEW

File: packages/editor/src/plugins/dropzone-image/components/answer-zone/answer-zone-answer.tsx [hunk 1]
```diff
@@ -21,12 +22,15 @@ interface AnswerZoneAnswerProps {
...
+  const store = useStore()
   const { answer, isOnlyAnswer, onEditAnswer, onRemoveAnswer } = props
 
   const pluginStrings = useEditStrings().plugins.dropzoneImage
 
-  const answerImageUrl = getAnswerZoneImageSrc(answer.image.id)
-  const answerText = getAnswerZoneText(answer.text.id)
+  const answerImageUrl = getAnswerZoneImageSrc(answer.image.id, () =>
+    store.getState()
+  )
```
Comment (CodingDive) [inline packages/editor/src/plugins/dropzone-image/components/answer-zone/answer-zone-answer.tsx:32]: I know that here I could technically just pass `store.getState` but to me, it's less clear that we are passing in a fn than using the arrow function syntax. Let me know which syntax you prefer

Comment (hejtful) [reply packages/editor/src/plugins/dropzone-image/components/answer-zone/answer-zone-answer.tsx:32]: This is fine with me.

In general, these utils shouldn't be calling the store state getter themselves, they should be provided only the relevant piece of state for better encapsulation...but that's another topic, unrelated to this PR 😁

File: packages/editor/src/store/store.ts [hunk 2]
```diff
@@ -1,38 +1,68 @@
...
+  sagaMiddleware.run(saga)
+
+  return store
+}
+
+// one store: Record<editorId, { state }>
+// multiple stores: one store per editor instance / per editor id
+
+// Export singleton instance of store
+// export const store = createStoreInstance()
```
Comment (hejtful) [inline packages/editor/src/store/store.ts]: Leftover from our call? 😁

**Review by hejtful — APPROVED** [review]
Clean and simple, thank you for the quick fix!

File: packages/editor/src/store/store.ts (hunk 2 above)
Comment (CodingDive) [reply packages/editor/src/store/store.ts]: Ahh nice catch! 😅

**Pull request merged**